indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -0,0 +1,659 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import fetchRoomURL from 'indico-url:rooms_new.admin_room';
+import fetchRoomAttributesURL from 'indico-url:rooms_new.admin_room_attributes';
+import fetchAttributesURL from 'indico-url:rooms_new.admin_attributes';
+import fetchRoomAvailabilityURL from 'indico-url:rooms_new.admin_room_availability';
+import fetchRoomEquipmentURL from 'indico-url:rooms_new.admin_room_equipment';
+import updateRoomBasicDetailsURL from 'indico-url:rooms_new.admin_update_room';
+import updateRoomEquipmentURL from 'indico-url:rooms_new.admin_update_room_equipment';
+import updateRoomAttributesURL from 'indico-url:rooms_new.admin_update_room_attributes';
+import updateRoomAvailabilityURL from 'indico-url:rooms_new.admin_update_room_availability';
+import _ from 'lodash';
+import React from 'react';
+import {bindActionCreators} from 'redux';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {Button, Checkbox, Dimmer, Dropdown, Form, Grid, Header, Input, Loader, Message, Modal, TextArea} from 'semantic-ui-react';
+import {Form as FinalForm, Field} from 'react-final-form';
+import {FieldArray} from 'react-final-form-arrays';
+import arrayMutators from 'final-form-arrays';
+import shortid from 'shortid';
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+import {snakifyKeys, camelizeKeys} from 'indico/utils/case';
+import {getChangedValues, ReduxCheckboxField, ReduxFormField} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+import PrincipalSearchField from 'indico/react/components/PrincipalSearchField';
```

Comment:
I am just wondering whether we should be consistent and add sth like `export {default as PrincipalSearchField} from './PrincipalSearchField'` in the `indico/react/components/index.js` which would allow to import the component without `components` part

Reply:
For this one yes, since we do have some things being re-reported there already